Source/WebCore/dom/NodeIterator.h

```diff
@@ -57,6 +57,7 @@ class NodeIterator final : public ScriptWrappable, public RefCounted<NodeIterato
 
         NodePointer() = default;
         NodePointer(Node&, bool);
+        RefPtr<Node> protectedNode() const { return node.copyRef(); }
```

Comment:
nit: we don't need the `.copyRef()` I believe.

Reply:
Sure, removed.